Develop an SQL schema for SOPs

```diff
@@ -1,29 +1,4351 @@
+    exp_mass_to_charge double precision NOT NULL,
+    calc_mass_to_charge double precision NOT NULL,
+    uri text NOT NULL,
+    spectra_ref text NOT NULL,
+    pre text NOT NULL,
+    post text NOT NULL,
+    start integer NOT NULL,
+    "end" integer NOT NULL
+);
+
+
+ALTER TABLE "MzTab"."PSM Section" OWNER TO postgres;
+
+--
+-- TOC entry 210 (class 1259 OID 17286)
+-- Name: Peptide Section; Type: TABLE; Schema: MzTab; Owner: postgres
+--
+
+CREATE TABLE "MzTab"."Peptide Section" (
+    sequence text NOT NULL,
+    accession text NOT NULL,
+    "unique" boolean NOT NULL,
+    database text NOT NULL,
+    database_version text NOT NULL,
+    search_engine text[] NOT NULL,
+    best_search_engine_score_number text NOT NULL,
+    best_search_engine_score double precision NOT NULL,
+    search_engine_score_number text NOT NULL,
+    ms_run_number text NOT NULL,
+    search_engine_score_ms_run double precision NOT NULL,
+    reliability integer NOT NULL,
+    modifications text NOT NULL,
+    retention_time double precision[] NOT NULL,
+    retention_time_window double precision[] NOT NULL,
+    charge integer NOT NULL,
+    mass_to_charge double precision NOT NULL,
+    uri text NOT NULL,
+    spectra_ref text NOT NULL,
+    peptide_abundance_assay_number text NOT NULL,
+    peptide_abundance_assay double precision NOT NULL,
+    peptide_abundance_study_variable_number text NOT NULL,
+    peptide_abundance_study_variable double precision NOT NULL,
+    peptide_abundance_stdev_study_variable_number text NOT NULL,
+    peptide_abundance_stdev_study_variable double precision NOT NULL,
+    peptide_abundance_std_error_study_variable_number text NOT NULL,
+    peptide_abundance_std_error_study_variable double precision NOT NULL,
+    id integer NOT NULL
+);
+
+
+ALTER TABLE "MzTab"."Peptide Section" OWNER TO postgres;
+
+--
+-- TOC entry 223 (class 1259 OID 18181)
+-- Name: Peptide Section_id_seq; Type: SEQUENCE; Schema: MzTab; Owner: postgres
+--
+
+CREATE SEQUENCE "MzTab"."Peptide Section_id_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE "MzTab"."Peptide Section_id_seq" OWNER TO postgres;
+
+--
+-- TOC entry 2967 (class 0 OID 0)
+-- Dependencies: 223
+-- Name: Peptide Section_id_seq; Type: SEQUENCE OWNED BY; Schema: MzTab; Owner: postgres
+--
+
+ALTER SEQUENCE "MzTab"."Peptide Section_id_seq" OWNED BY "MzTab"."Peptide Section".id;
+
+
+--
+-- TOC entry 209 (class 1259 OID 17263)
+-- Name: Protein section; Type: TABLE; Schema: MzTab; Owner: postgres
+--
+
+CREATE TABLE "MzTab"."Protein section" (
+    accession text NOT NULL,
+    description text NOT NULL,
+    taxid integer NOT NULL,
+    species text NOT NULL,
+    database text NOT NULL,
+    database_version text NOT NULL,
+    search_engine text[] NOT NULL,
+    best_search_engine_score_number text NOT NULL,
+    best_search_engine_score text NOT NULL,
+    search_engine_score_number text NOT NULL,
```

Comment:
Should these scores also be integer or double?

Reply:
done

Reply:
done